src/foam/u2/dialog/StyledModal.js

```diff
@@ -134,7 +134,7 @@ foam.CLASS({
           .start()
             .enableClass(this.myClass('inner'), this.isStyled$)
             .startContext({ data: this })
-              .start(this.CLOSE_MODAL, { buttonStyle: 'TERTIARY' }).show(this.closeable$)
+              .start(this.CLOSE_MODAL, { buttonStyle: 'TERTIARY', label: '' }).show(this.closeable$)
```

Comment:
Why don't we remove this from the modal causing the issue?

Reply:
do you mean popup?